src/vm/codeman.cpp

```diff
@@ -6898,50 +6898,56 @@ BOOL ReadyToRunJitManager::JitCodeToMethodInfo(RangeSection * pRangeSection,
     // READYTORUN: FUTURE: Hot-cold spliting
 
     TADDR currentInstr = PCODEToPINSTR(currentPC);
+    TADDR imageBase = pRangeSection->LowAddress;
 
-    TADDR ImageBase = pRangeSection->LowAddress;
-
-    DWORD RelativePc = (DWORD)(currentInstr - ImageBase);
+    DWORD relativePc = (DWORD)(currentInstr - imageBase);
 
     Module * pModule = dac_cast<PTR_Module>(pRangeSection->pHeapListOrZapModule);
     ReadyToRunInfo * pInfo = pModule->GetReadyToRunInfo();
 
     COUNT_T nRuntimeFunctions = pInfo->m_nRuntimeFunctions;
     PTR_RUNTIME_FUNCTION pRuntimeFunctions = pInfo->m_pRuntimeFunctions;
 
-    int MethodIndex = NativeUnwindInfoLookupTable::LookupUnwindInfoForMethod(RelativePc,
+    int methodIndex = NativeUnwindInfoLookupTable::LookupUnwindInfoForMethod(relativePc,
                                                                              pRuntimeFunctions,
                                                                              0,
                                                                              nRuntimeFunctions - 1);
 
-    if (MethodIndex < 0)
+    if (methodIndex < 0)
+    {
         return FALSE;
+    }
+
+    if (ppMethodDesc == NULL && pCodeInfo == NULL)
+    {
+        // Bail early if caller doesn't care about the MethodDesc or EECodeInfo.
```

Comment:
It would be useful to add a comment here that this is also required to avoid deadlocks when this is called from IsManagedCode.